src/transactions/feed/queryHelper.ts

```diff
@@ -150,16 +151,20 @@ export function useFetchTransactions(): QueryHookResult {
 
   const [fetchingMoreTransactions, setFetchingMoreTransactions] = useState(false)
 
-  // Update the counter variable every |POLL_INTERVAL| so that a query is made to the backend.
-  const [counter, setCounter] = useState(0)
-  useInterval(() => setCounter((n) => n + 1), POLL_INTERVAL)
+  useEffect(() => {
+    const id = setInterval(() => {
+      void pollNewTransactions.execute()
+    }, POLL_INTERVAL)
+
+    return () => clearInterval(id)
+  }, [])
 
   const handleError = (error: Error) => {
     Logger.error(TAG, 'Error while fetching transactions', error)
   }
 
   // Query for new transaction every POLL_INTERVAL
-  const { loading, error } = useAsync(
+  const pollNewTransactions = useAsyncCallback(
```

Comment:
this isn't a consequential change, but i think it is an improvement on readability + removing an unnecessary state variable